OpenRA.Mods.Common/Traits/Buildings/ProductionAirdrop.cs

```diff
@@ -53,29 +54,31 @@ public override bool Produce(Actor self, ActorInfo producee, string productionTy
 			var aircraftInfo = self.World.Map.Rules.Actors[info.ActorType].TraitInfo<AircraftInfo>();
 			var mpStart = owner.World.WorldActor.TraitOrDefault<MPStartLocations>();
```

Comment:
Can we please now remove this and instead use `owner.HomeLocation` if BaselineSpawn is enabled?

This will fix baseline production in scripted maps.